src/stk/cage.py

```diff
@@ -22,6 +22,7 @@
 from stk._internal.topology_graphs.cage.m4l8 import M4L8
 from stk._internal.topology_graphs.cage.m6l2l3_prism import M6L2L3Prism
 from stk._internal.topology_graphs.cage.m6l12_cube import M6L12Cube
+from stk._internal.topology_graphs.cage.m6l6l12_cuboctahedron import M6L6L12Cuboctahedron
```

Comment:
should this be `cage.m6l8`? I think thats the cause of the failing pytest

Reply:
yes, that was fixes with the new commit that i just pushed. I put the wrong names the first time and fixed them (hopefully everywhere) now